plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/jdbc/TestIcebergJdbcCatalogConnectorSmokeTest.java

```diff
@@ -46,6 +52,7 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     protected QueryRunner createQueryRunner()
             throws Exception
     {
+        warehouseLocation = Files.createTempDirectory("test_iceberg_jdbc_catalog_smoke_test").toFile();
```

Comment:
Do delete the warehouse directory after the tests.

https://github.com/trinodb/trino/blob/9f1e41bfadca8c909c89986cddea5e7513bd679a/plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergQueryRunner.java#L157